ui/pages/confirm-transaction-base/confirm-transaction-base.test.js

```diff
@@ -347,8 +350,13 @@ describe('Confirm Transaction Base', () => {
       />,
       store,
     );
+
     const confirmButton = getByTestId('page-container-footer-next');
-    fireEvent.click(confirmButton);
+
+    await act(async () => {
+      fireEvent.click(confirmButton);
```

Comment:
This may have implications higher up the component stack if we want to run multiple handlers strictly sequentially, but this is the initial quick fix to resolve the pipeline failure.